packages/core-mobile/app/components/EditFees.tsx

```diff
@@ -91,7 +91,7 @@ const EditFees = ({
    * EVM - gWei
    */
   const [newMaxFeePerGas, setNewMaxFeePerGas] = useState<string>(
-    bigIntToFeeDenomination(initMaxFeePerGas, isBtcNetwork)
+    bigIntToFeeDenomination(initMaxFeePerGas, isSimpleFeeRate)
```

Comment:
I think it's a bit confusing to include isSimpleFeeRate in bigIntToFeeDenomination. it's not really a responsibility of bigIntToFeeDenomination to know about that. also what is isSimpleFeeRate really? from the code it looks like it's already in the right unit and doesn't need any conversion.

Reply:
maybe renaming it to something else like `isPreConvertedRate`, `skipFeeConversion`, `isBaseUnitRate`?